roles/ipaclient/vars/CentOS-7.yml

```diff
@@ -1,4 +1,5 @@
 # defaults file for ipaclient
 # vars/rhel.yml
 ipaclient_packages: [ "ipa-client", "libselinux-python" ]
+gssapi_packages: [ "python2-gssapi.x86_64" ]
```

Comment:
Is the architecture needed?
Why not just use:

gssapi_packages: ["python2-gssapi"]

Reply:
The architecture should not be used here as this would only work on x86_64.